__mocks__/chart-widgets-config.mock.ts

```diff
@@ -10,6 +10,17 @@ export const ConfigMock = {
   },
   vitals: {
     formUuid: "a000cb34-9ec1-4344-a1c8-f692232f6edd",
-    encounterTypeUuid: "67a71486-1a54-468f-ac3e-7091a9a79584"
+    encounterTypeUuid: "67a71486-1a54-468f-ac3e-7091a9a79584",
+    bloodPressureUnit: "mmHg",
+    oxygenSaturationUnit: "%",
+    pulseUnit: "bpm",
+    respiratoryRateUnit: "/min",
+    temperatureUnit: "°C",
+    midUpperArmCircumferenceUnit: "cm"
+  },
+  biometrics: {
+    bmiUnit: "kg / m²",
+    heightUnit: "cm",
+    weightUnit: "kg"
```

Comment:
All these units except the bmiUnit should come from the corresponding concept metadata.

Reply:
I don't know the the are coded via metadata. I think they are part of the concept itself. Such that weight in lb's would use a a different than weight in kgs. I just looked in the openmrs concept dictionary and there doesn't seem to be any way to infer units from these concepts other than to just "know" what the right one is.  @donaldkibet you probably have a dictionary object that maps concepts to their corresponding units. If someone were to use the weight concept (for example), you would be displaying the wrong units.

Reply:
It's definitely part of the model:

![screenshot--2020-12-08--102332](https://user-images.githubusercontent.com/1031876/101524964-80b4fd80-393f-11eb-905c-0e4e5d6c7b27.png)

Reply:
I stand corrected